### DISCUSSION TITLE
is it possible to hide or change the computer details per sandbox window?

### DISCUSSION THREAD

**Discussion by akkocton** [sandboxie-plus/Sandboxie#3068] [General]
Thank you, what should i type at edit.ini

**Comment by bastik-1001** [comment]
As far as I am aware, there isn't much that could be hidden. There is a [feature request](https://github.com/sandboxie-plus/Sandboxie/issues/1712) for something like that.

You can use `HideHostProcess=process.exe` to hide host processes, by specifying the name of the process, from the application inside the box. e.g. HideHostProcess=notepad.exe

**Reply by akkocton** [reply]
Thanks for the response, looks like, gonna wait for the future request to be granted.

**Reply by akkocton** [reply]
and sir sorry if im asking this much, maybe you have an idea to set different ip per box?

**Reply by akkocton** [reply]
does per window box have his own .EXE file? so i could connect it to some app like proxifier? ty

**Reply by bastik-1001** [reply]
[The roadmap](https://www.wilderssecurity.com/threads/sandboxie-roadmap.445545/) contains an upcoming feature:
    
> added ability to for[c]e sandboxed processes to use a pre defined socks 5 proxy
> added ability to intercept DNS queries such that thay can be log and/or redirected

So whenever some proxy allows you to bind to two (or more) different IP addresses, this might work. I am not sure if that works on a technical level, but whenever some software proxy has such a feature it should be possible to use it.

I am not entirely sure, what you are asking for with:

>  does per window box have his own .EXE file? [...]

but when you start program.exe in box A and in box B, at the same time. Those will be different processes, each with their own process ID (PID).

**Reply by akkocton** [reply]
Thank you for clarification again ,so much appreciated.

gonna wait for the new update

since sir thanos spoil it when to release it..

Immediate plans for the upcoming builds:

1.10.0 (this week)
Will get a new [troubleshooting ](https://www.wilderssecurity.com/threads/sandboxie-plus-troubleshooting-wizard-upcomming-new-feature.451560/)wizard based on the new [box script engine](https://www.wilderssecurity.com/threads/sandboxie-scripting-engine.451576/) the troubleshooting scripts will be able of being updates online to always get the latest once, and it has a feature to submit issue reports through the wizard.

1.11.0 (in a few weeks)
Will get the ability to create [ram disk based sandboxes](https://sandboxie-plus.com/ramdisksupport/) users with access to the [insider program](https://sandboxie-plus.com/ramdisksupport/) can already use this feature and many more.
